src/contracts/version/Registry.sol

```diff
@@ -82,12 +85,46 @@ contract Registry is DSAuth {
 
     // PUBLIC METHODS
 
-    function registerFund(address _fund) {
+    /// @notice Whether _name has only valid characters
+    function isValidFundName(string _name) public view returns (bool) {
+        bytes memory b = bytes(_name);
+        if (b.length > MAX_FUND_NAME_BYTES) return false;
+        for (uint i; i < b.length; i++){
+            bytes1 char = b[i];
+            if(
+                !(char >= 0x30 && char <= 0x39) && // 9-0
+                !(char >= 0x41 && char <= 0x5A) && // A-Z
+                !(char >= 0x61 && char <= 0x7A) && // a-z
+                !(char == 0x20 || char == 0x2D) && // space, dash
+                !(char == 0x2E || char == 0x5F)    // period, underscore
```

Comment:
can we add also `|| char  == 0x2A` for *  (line 99)  @travs

Reply:
0650ae883b71407f437f51a6ae2b77426daa614d